server/tm_cmds.go

```diff
@@ -59,6 +85,37 @@ func ShowValidatorCmd(ctx *Context) *cobra.Command {
 	return &cmd
 }
 
+//This is a copy of a the same function from Tendermint.
```

Comment:
Could we import it from Tendermint then?

Reply:
For that it would be needed to be exported/public first.

Reply:
++

Reply:
Yes, you are probably right. If this is the only use-case (return the remote signer's pub-key), we might consider adding a simple method that does exactly this instead. I thought `privValidator.GetPubKey()` does exactly this (as it sends a request in the remote case and returns what was read from the file in the `FilePV` case). Have not tried if this works though.

Reply:
Sounds great!